.github/workflows/build.yml

```diff
@@ -133,12 +133,15 @@ jobs:
             - uses: actions/setup-python@v3
               with:
                 python-version: '3.7'
+            - uses: ilammy/msvc-dev-cmd@v1
```

Comment:
Runs the windows developer command prompt which is needed for Ninja Win32 builds

https://github.com/ilammy/msvc-dev-cmd